docs/develop-component.md

```diff
@@ -42,7 +42,11 @@ For an example of this check [test/.../docs/test-component-with-duplicate-ids.ym
 
 ## Visual regression testing
 
-The component guide includes a visual diff tool that should make it easier to spot when you are introducing visual regressions in your components.
+Visual regression tests are run on each pull request using a third-party tool called [Percy](https://percy.io). This isn't a mandatory check, so any changes that are highlighted by Percy won't prevent a pull request from being merged in.
```

Comment:
Currently only the `continuous-integration/jenkins/branch` is a required check that blocks merging. This can be changed in the future - but whilst there's the danger of running out of credits and the check not being able to be completed I think it should remain optional.

Reply:
Hmm.. As I see it at this very moment I'm unable to merge the following PR https://github.com/alphagov/govuk_publishing_components/pull/2016 until someone reviews it in Percy.